hack/deploy/controlplane/external-etcd-patch.yaml

```diff
@@ -38,14 +38,14 @@ spec:
           - "--external-hostname=API_HOST"
         volumeMounts:
         - $patch: delete
-          mountPath: /.ocmconfig
-          name: ocmconfig 
+          mountPath: /.embedded-etcd 
```

Comment:
Do we want to change this path? AFAIK, the default embed etcd path is `/.ocmconfig`: https://github.com/open-cluster-management-io/multicluster-controlplane/blob/main/pkg/servers/options/embeddedetcd.go#L37

Reply:
Here is only for patch. if we configure to use external etcd, we should delete `embedded-etcd` which is using in https://github.com/stolostron/multicluster-controlplane/blob/c5f53897ef130abff02f319ad3d7a7bcfeac333c/hack/deploy/controlplane/deployment.yaml#L68. Here is a bug we cannot delete the PV volume w/ external etcd.